File: app/src/testShared/bj/discogsbrowser/artist/ArtistFactory.java
package bj.discogsbrowser.artist;

import bj.discogsbrowser.model.common.Artist;

/**
 * Created by Josh Laird on 16/05/2017.
 */

public class ArtistFactory
{
    public static Artist buildSingleArtist()
    {
        Artist artist = new Artist();
        artist.setName("ye son");
        return artist;
    }
}

File: app/src/testShared/bj/discogsbrowser/artist/ArtistFactory.kt
package bj.discogsbrowser.artist

import bj.discogsbrowser.model.common.Artist

/**
 * Created by Josh Laird on 19/05/2017.
 */
class ArtistFactory {
    companion object {
        @JvmStatic fun buildArtist(): Artist {
            return Artist(name = "ye son")
        }
    }
}